lib/App/Yath/Command/runner.pm

```diff
@@ -374,6 +374,11 @@ sub update_io {
         my $msg = "$_[0] at $caller[1] line $caller[2] ($caller2[1] line $caller2[2]).\n";
         print $stderr $msg;
         print STDERR $msg;
+        eval {
```

Comment:
This eval is not necessary, close() is not fatal when it fails. IF we do keep the eval I would like $@ localized just before the eval.

Reply:
fair enough let's remove the eval :-)

Reply:
updated via 3946062e5efa2536b70530bc0a2c71df74cf4fda